test/integration/default/default_test.rb

```diff
@@ -77,7 +77,11 @@
   its('mailbox_command') { should eq '/usr/bin/procmail' }
   its('mailbox_size_limit') { should eq '0' }
   its('message_size_limit') { should eq '102400000' }
-  its('transport_maps') { should eq 'hash:/etc/postfix/transport' }
+  if os.release.to_i < 10
+    its('transport_maps') { should eq 'hash:/etc/postfix/transport' }
+  else
+    its('transport_maps') { should eq 'texthash:/etc/postfix/transport' }
```

Comment:
This test is failing for me for some reason.

Reply:
So here's the problem, on EL10 the hash DB type has been deprecated in favor of another one. You need to change this line:

https://github.com/osuosl-cookbooks/osl-rt/blob/4e939dd0f050dc92211307cf152e838de6b7357f/recipes/default.rb#L27

To use `node['postfix']['db_type']` instead of `hash`. You'll need to use double quotes to bring that in. The upstream postfix cookbook has logic to set it properly. You'll need to fix this test to use `lmdb` instead of `hash` on EL10.